[2021-11-17T19:37:57.259373+00:00] php.DEBUG: Warning: filemtime(): stat failed for /home/csattoszsolt/MyProjects/Trendency/local/skeleton/ecosystem/cms/backend/config/routes/dev/web_profiler.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/app/vendor/symfony/config/Resource/FileResource.php","line":62,"trace":[{"file":"/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-11-17T19:37:57.309817+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-17T19:37:57.488591+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:37:57.511627+00:00] php.DEBUG: Warning: filemtime(): stat failed for /home/csattoszsolt/MyProjects/Trendency/local/skeleton/ecosystem/cms/backend/config/routes/dev/web_profiler.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/app/vendor/symfony/config/Resource/FileResource.php","line":62,"trace":[{"file":"/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-11-17T19:37:57.565995+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:37:57.566630+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:37:57.569050+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-17T19:37:57.698210+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-17T19:37:57.698377+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T08:10:47.043815+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T08:10:47.188542+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T08:10:47.233228+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T08:10:47.233508+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T08:10:47.234154+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T08:10:47.356122+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T08:10:47.356252+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:52.779624+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T13:59:52.986174+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:53.032031+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:53.032215+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:53.033042+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:53.162030+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:53.162180+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:55.671056+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T13:59:55.813340+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:55.859736+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:55.859924+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:55.860457+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:55.993725+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:55.993861+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:56.475422+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T13:59:56.629314+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:56.677651+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:56.677852+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:56.678603+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:56.814295+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:56.814456+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:56.988667+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-18T13:59:57.139874+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/")" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:57.190914+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:57.191103+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:57.191749+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-18T13:59:57.331702+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-18T13:59:57.331967+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/\") at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:27:57.768671+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.localhost/ws"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/ws\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/ws/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:27:57.876829+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T12:27:57.877297+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/ws\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/ws/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:27:57.879243+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in the driver: could not find driver" at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/ws\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/ws/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:27:58.019254+00:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 114) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)"} []
[2021-11-20T12:27:58.019444+00:00] php.CRITICAL: Uncaught Exception: An exception occurred in the driver: could not find driver {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:114)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /app/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:34)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/ws\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/ws/\". at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-11-20T12:28:32.678300+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T12:44:09.118814+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-11-20T13:17:14.822512+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://api.cms.localhost/"" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://api.cms.localhost/\" at /app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
